src/test/java/com/exasol/config/ClusterConfigAccessIT.java

```diff
@@ -25,4 +25,12 @@ void testGetClusterConfiguration() {
         assertThat(this.clusterConfiguration.getBucketFsServiceConfiguration("bfsdefault").getName(),
                 equalTo("bfsdefault"));
     }
+
+    @Test
+    void testGetAuthenticationToken() {
+        this.clusterConfiguration = CONTAINER.getClusterConfiguration();
+        final String token = this.clusterConfiguration.getDefaultAuthenticationToken();
+        assertThat(token, not(emptyOrNullString()));
+        assertThat(token.length(), greaterThan(20));
```

Comment:
That test is a bit weak. But doing a login is too much effort here?

Reply:
I could use the bucketfs-java lib to call a method. Or I could do the call manually here. I will try.